src/Sarif.Multitool/FileWorkItemsCommand.cs

```diff
@@ -28,89 +35,56 @@ public int Run(FileWorkItemsOptions options)
 
         public int Run(FileWorkItemsOptions options, IFileSystem fileSystem)
         {
-            if (!ValidateOptions(options, fileSystem)) { return FAILURE; }
+            var filingContext = new SarifWorkItemContext();
+            if (!string.IsNullOrEmpty(options.ConfigurationFilePath))
+            {
+                filingContext.LoadFromXml(options.ConfigurationFilePath);
+            }
+
+            if (!ValidateOptions(options, filingContext, fileSystem)) 
+            { 
+                return FAILURE; 
```

Comment:
>FAILURE [](start = 23, length = 7)

Just checking -- you definitely want to return `FAILURE` here, despite the fact that an individual option might override the faulty option in the configuration file and make it valid?

Reply:
Good question. But this helper validates the combination of command-line options and configuration, so there's no subsequent opportunity for things to be brought into a valid state. If you look, you'll see that the only remaining processing is to move various enum values into the configuration, if they exist.

---
In reply to: [381015408](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381015408) [](ancestors = 381015408)